### ISSUE TITLE
docker error - Unknown lvalue 'TasksMax' in section 'Service'

### ISSUE THREAD

**Issue by miabbott** [CentOS/sig-atomic-buildscripts#226]
Saw this today when testing latest CAHC.  I spoke with @lsm5 on #atomic and he said he was working on a fix.

```
$ sudo journalctl -u docker -b
-- Logs begin at Tue 2017-01-03 17:01:19 UTC, end at Wed 2017-01-11 19:49:22 UTC. --
Jan 11 19:47:17 cahc-dev systemd[1]: [/usr/lib/systemd/system/docker.service:16] Unknown lvalue 'TasksMax' in section 'Service'

$ rpm -q docker
docker-1.12.5-15.git079fbe3.el7.x86_64

$ atomic host status
State: idle
Deployments:
● centos-atomic-continuous:centos-atomic-host/7/x86_64/devel/continuous
       Version: 7.2017.17 (2017-01-11 19:08:58)
        Commit: a28e2f7aa6a5f29cf6f23361afd26f18dc2ace4fd707f63f8af2f48aacde0596
        OSName: centos-atomic-host

  centos-atomic-continuous:centos-atomic-host/7/x86_64/devel/continuous
       Version: 7.2017.11 (2017-01-09 18:15:03)
        Commit: 68b594696d7677a82ae6deed6fad1b4f5f7f38535b725dbeab2841f38178841c
        OSName: centos-atomic-host
```

**Comment by lsm5** [comment]
should be fixed with https://cbs.centos.org/koji/taskinfo?taskID=152318

On Wed, Jan 11, 2017 at 12:36:09PM -0800, Micah Abbott wrote:
> Saw this today when testing latest CAHC.  I spoke with @lsm5 on #atomic and he said he was working on a fix.
> 
> ```
> $ sudo journalctl -u docker -b
> -- Logs begin at Tue 2017-01-03 17:01:19 UTC, end at Wed 2017-01-11 19:49:22 UTC. --
> Jan 11 19:47:17 cahc-dev systemd[1]: [/usr/lib/systemd/system/docker.service:16] Unknown lvalue 'TasksMax' in section 'Service'
> 
> $ rpm -q docker
> docker-1.12.5-15.git079fbe3.el7.x86_64
> 
> $ atomic host status
> State: idle
> Deployments:
> ● centos-atomic-continuous:centos-atomic-host/7/x86_64/devel/continuous
>        Version: 7.2017.17 (2017-01-11 19:08:58)
>         Commit: a28e2f7aa6a5f29cf6f23361afd26f18dc2ace4fd707f63f8af2f48aacde0596
>         OSName: centos-atomic-host
> 
>   centos-atomic-continuous:centos-atomic-host/7/x86_64/devel/continuous
>        Version: 7.2017.11 (2017-01-09 18:15:03)
>         Commit: 68b594696d7677a82ae6deed6fad1b4f5f7f38535b725dbeab2841f38178841c
>         OSName: centos-atomic-host
> ```
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/CentOS/sig-atomic-buildscripts/issues/226

-- 
Lokesh
Freenode: lsm5
GPG: 0xC7C3A0DD
https://keybase.io/lsm5

**Comment by miabbott** [comment]
@lsm5 Looks like we need the same for `docker-containerd`

```
Jan 17 22:07:08 cahc-dev systemd[1]: [/usr/lib/systemd/system/docker-containerd.service:8] Unknown lvalue 'TasksMax' in section 'Service'
```

```
# rpm -q docker
docker-1.12.6-11.git51ef5a8.el7.x86_64
# cat /usr/lib/systemd/system/docker-containerd.service
[Unit]
Description=Containerd Standalone OCI Container Daemon
Documentation=https://containerd.tools/
After=network.target

[Service]
ExecStart=/usr/libexec/docker/docker-containerd-current --listen unix:///run/containerd.sock --shim /usr/libexec/docker/docker-containerd-shim-current --start-timeout 2m
TasksMax=8192
LimitNOFILE=1048576
LimitNPROC=1048576
LimitCORE=infinity

[Install]
WantedBy=multi-user.target
```

**Comment by lsm5** [comment]
On Tue, Jan 17, 2017 at 02:12:06PM -0800, Micah Abbott wrote:
> @lsm5 Looks like we need the same for `docker-containerd`

Sorry about that, I'll fix this tonight.

> 
> ```
> Jan 17 22:07:08 cahc-dev systemd[1]: [/usr/lib/systemd/system/docker-containerd.service:8] Unknown lvalue 'TasksMax' in section 'Service'
> ```
> 
> ```
> # rpm -q docker
> docker-1.12.6-11.git51ef5a8.el7.x86_64
> # cat /usr/lib/systemd/system/docker-containerd.service
> [Unit]
> Description=Containerd Standalone OCI Container Daemon
> Documentation=https://containerd.tools/
> After=network.target
> 
> [Service]
> ExecStart=/usr/libexec/docker/docker-containerd-current --listen unix:///run/containerd.sock --shim /usr/libexec/docker/docker-containerd-shim-current --start-timeout 2m
> TasksMax=8192
> LimitNOFILE=1048576
> LimitNPROC=1048576
> LimitCORE=infinity
> 
> [Install]
> WantedBy=multi-user.target
> ```
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/CentOS/sig-atomic-buildscripts/issues/226#issuecomment-273316930

-- 
Lokesh
Freenode: lsm5
GPG: 0xC7C3A0DD
https://keybase.io/lsm5

**Comment by lsm5** [comment]
On Tue, Jan 17, 2017 at 05:24:11PM -0500, Lokesh Mandvekar wrote:
> On Tue, Jan 17, 2017 at 02:12:06PM -0800, Micah Abbott wrote:
> > @lsm5 Looks like we need the same for `docker-containerd`
> 
> Sorry about that, I'll fix this tonight.

Should be all set
> 
> > 
> > ```
> > Jan 17 22:07:08 cahc-dev systemd[1]: [/usr/lib/systemd/system/docker-containerd.service:8] Unknown lvalue 'TasksMax' in section 'Service'
> > ```
> > 
> > ```
> > # rpm -q docker
> > docker-1.12.6-11.git51ef5a8.el7.x86_64
> > # cat /usr/lib/systemd/system/docker-containerd.service
> > [Unit]
> > Description=Containerd Standalone OCI Container Daemon
> > Documentation=https://containerd.tools/
> > After=network.target
> > 
> > [Service]
> > ExecStart=/usr/libexec/docker/docker-containerd-current --listen unix:///run/containerd.sock --shim /usr/libexec/docker/docker-containerd-shim-current --start-timeout 2m
> > TasksMax=8192
> > LimitNOFILE=1048576
> > LimitNPROC=1048576
> > LimitCORE=infinity
> > 
> > [Install]
> > WantedBy=multi-user.target
> > ```
> > 
> > -- 
> > You are receiving this because you were mentioned.
> > Reply to this email directly or view it on GitHub:
> > https://github.com/CentOS/sig-atomic-buildscripts/issues/226#issuecomment-273316930
> 
> -- 
> Lokesh
> Freenode: lsm5
> GPG: 0xC7C3A0DD
> https://keybase.io/lsm5



-- 
Lokesh
Freenode: lsm5
GPG: 0xC7C3A0DD
https://keybase.io/lsm5

**Comment by miabbott** [comment]
This appears fixed in the latest CAHC commits (7.2017.41)
